gers/bridge_file/bridge_file_schema.yaml

```diff
@@ -0,0 +1,11 @@
+"$schema": https://json-schema.org/draft/2020-12/schema
```

Comment:
Suggest that we replace this with another representation of the schema table-- parquet? duckdb view?